plugins/single_plugins/switcher.cpp

```diff
@@ -128,12 +128,16 @@ class WayfireSwitcher : public wf::plugin_interface_t
         touch_activate = [=] (wf_touch_gesture*) {
             if (!active)
             {
+                if (!output->can_activate_plugin(grab_interface))
```

Comment:
I still don't get why we need this check instead of returning the return value of handle_switch_request() directly ...

Reply:
I see now.